### ISSUE TITLE
Find trait method implementations

### ISSUE THREAD

**Issue by dbrgn** [intellij-rust/intellij-rust#2122]
I hope this wasn't reported before, I didn't find an issue for it.

On a trait, it's possible to list all impls:

![img](https://tmp.dbrgn.ch/screenshots/20171212111004-n_fbog0l.png)

But this isn't available on a trait method:

![img](https://tmp.dbrgn.ch/screenshots/20171212111032-ygychlkd.png)

If I'm not mistaken, IntelliJ supports this for abstract methods in Java. Would be awesome to have this in Rust too :)

**Comment by dbrgn** [comment]
Interesting: The Rust plugin correctly detects that the implementing method implements this trait:

![img](https://tmp.dbrgn.ch/screenshots/20171212111232-vsdca_61.png)

So it's possible to navigate up in the hierarchy, but not back down.

**Comment by kumbayo** [comment]
I am looking into this (already have a prototype)

**Comment by tchalamarla** [comment]
while at it, is it possible to generate code for all functions if 
we type
impl From<u8> for X {
}

can it place dummy functions like
impl From<u8> for X {
    fn from<u8>(d: u8) -> Self {
          unreachable!() /unimplemented!()
    }
}


sorry if this is not the right place for this

**Comment by kumbayo** [comment]
@tchalamarla This feature sort of already exists.
Just write 
```rust
impl From for X {
}
```
The first line will be underlined with a red squiggly line to mark missing items.
Just press Alt-Enter and choose `implement members`
